sphinx_gallery/gen_gallery.py

```diff
@@ -505,81 +505,84 @@ def generate_gallery_rst(app):
         costs += this_costs
         write_computation_times(gallery_conf, gallery_dir_abs_path, this_costs)
 
-        # We create an index.rst with all examples
-        # (this will overwrite the rst file generated by the previous call
-        # to generate_dir_rst)
-        index_rst_new = os.path.join(gallery_dir_abs_path, 'index.rst.new')
-        with codecs.open(index_rst_new, 'w', encoding='utf-8') as fhindex:
-            # :orphan: to suppress "not included in TOCTREE" sphinx warnings
-            fhindex.write(":orphan:\n\n" + this_content)
-
-            # Write toctree with gallery items from gallery root folder
-            if len(this_toctree_items) > 0:
-                this_toctree = """
+        if this_content:
```

Comment:
Everything from here to the `else` is just an indentation change of the original code...

Reply:
Github "Hide whitespace " viewing mode agrees :)